bindings/go/opendal.go

```diff
@@ -181,7 +181,10 @@ func NewOperator(scheme Scheme, opts OperatorOptions) (op *Operator, err error)
 	optionsFree := getFFI[operatorOptionsFree](ctx, symOperatorOptionsFree)
 
 	for key, value := range opts {
-		setOptions(options, key, value)
+		err = setOptions(options, key, value)
+		if err != nil {
+			return
```

Comment:
Is  ` defer optionsFree(options)` still needs to call here?

Reply:
Good point! Fixed.